src/components/Common/Button/Button.tsx

```diff
@@ -9,6 +9,7 @@ interface ButtonSpec {
 	children: ReactNode,
 	onClick?: React.MouseEventHandler<HTMLButtonElement>,
 	disabled?: boolean,
+	fullHeigth?: boolean,
```

Comment:
This is not how "height" is supposed to be written...

Reply:
well